intranet/apps/polls/tests.py

```diff
@@ -57,6 +57,9 @@ def test_poll(self):
         user_choice = Answer.objects.get(question=Question.objects.get(poll=poll[0]), user=user).choice
```

Comment:
These tests look good. However, can you also add a test before the `poll_vote` request above?